portfolio/src/main/webapp/index.html

```diff
@@ -65,9 +66,21 @@ <h2>My Favourite Food</h2>
         <button class="next-slide-button">Next</button>
       </div>
     
-      <h2>Comments</h2>
+      <h2 id="comment-section">Comments</h2>
       <div id="comment-container"></div>
 
+      <form action="/data" method="POST">
+        <label for="name">Name:</label><br>
+        <textarea name="name-input">Full Name</textarea>
+        <br/><br/>
+
+        <label for="comment">Comment:</label><br>
```

Comment:
What does the label element do in this case? Same above with name.

Reply:
Do you mean the use of the label tag instead of something else, or the redundancy in asking for Name and then Full name in the textarea?

I have removed the text in the textarea to fix the redundancy. I tried to do it by amending my previous commit but I'm not sure if I did it right since I had to resolve some conflicts after.

I used label instead of say, <p>, because I found this resource: https://www.w3schools.com/tags/tag_label.asp which said label is better for screen reader users. If <p> is preferred though I could change it?

Reply:
You addressed my question with the screen reader. Thank you.